package.json

```diff
@@ -53,6 +53,7 @@
         "@ourworldindata/utils": "workspace:^",
         "@prettier/plugin-php": "^0.14.3",
         "@react-awesome-query-builder/antd": "^6.1.1",
+        "@sinclair/typebox": "^0.28.5",
```

Comment:
I haven't used this one before. Seems cool!

Seeing as it has a lot of potential usecases throughout our project, maybe it's worth a quick demo to the rest of the team?